source/_integrations/twentemilieu.markdown

```diff
@@ -18,22 +18,123 @@ ha_platforms:
 ha_integration_type: service
 ---
 
-The Twente Milieu integration allows you to track the next scheduled waste
-pickups by Twente Milieu for each of the different waste types.
+The Twente Milieu {% term integration %} enables you to monitor the upcoming
+waste collection schedules provided by
+[Twente Milieu](https://www.twentemilieu.nl/) for various waste categories.
+This integration helps you stay informed about the next pickup dates for
+different types of waste, ensuring you never miss a collection day.
 
 {% include integrations/config_flow.md %}
 
-## Calendar
+### Configuration parameters
+
+{% configuration_basic %}
+Postal code:
+  description: The postal code of the address, for example "7500AA".
+House number:
+  description: The house number of the address.
+House letter/additional:
+  description: The house letter or additional information of the address.
+{% endconfiguration_basic %}
+
+## Use cases
+
+The integration provides sensors for the next waste pickup dates. You can use
+this information to create automations, for example, to remind you to put out
+the waste bins the night before the pickup.
+
+Besides the sensors, the integration also provides a calendar to Home Assistant.
+Meaning you can view all upcoming waste pickups in the calendar dashboard.
+
+## Supported functionality
+
+### Calendar
 
 The integration provides a calendar to Home Assistant. You can view
 all upcoming waste pickups in the calender dashboard.
 
-## Sensors
+### Sensors
 
-This integration provides sensors for the following waste pickup dates from Twente Milieu:
+This integration provides sensors for the following waste pickup dates from
+Twente Milieu:
 
 - Next plastic waste pickup date.
 - Next organic waste pickup date.
 - Next paper waste pickup date.
 - Next non-recyclable waste pickup date.
 - Next Christmas Tree pickup date.
+
+## Data updates
+
+The integration will update its information by polling Twente Milieu every
+hour. This ensures the data in Home Assistant is up to date.
+
+## Actions
+
+This integration does not provide additional actions.
```

Comment:
If it's not there should we document it here? Mainly a question for how to review for other integrations.

Reply:
The IQS doesn't document an exception, so from my perspective, there isn't really an exception.

From the user perspective, this removes the question if the integration provides additional actions or not 🤷